VK_App/VK_App/Helpers/DataReloadable.swift

```diff
@@ -0,0 +1,9 @@
+// DataReloadable.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Методы для перезагрузки UITableView
```

Comment:
```suggestion
/// Метод для обновления ячеек
```
он у нас тут один и может использоваться не только для обновления ячеек UITableView